apps/remote_control/lib/lexical/remote_control/search/store/backend.ex

```diff
@@ -11,18 +12,29 @@ defmodule Lexical.RemoteControl.Search.Store.Backend do
           subtypes: [Entry.entry_subtype()]
         }
 
-  @callback new(Path.t()) :: {:ok, load_state(), name()} | {:error, :any}
-  @callback new(Path.t(), version()) :: {:ok, load_state(), name()} | {:error, any()}
+  @callback new(Project.t()) :: {:ok, load_state()} | {:error, :any}
```

Comment:
Is this `:any` meant to be `any()`?

```suggestion
  @callback new(Project.t()) :: {:ok, load_state()} | {:error, any()}
```